functional/fixtures/units/global@.service

```diff
@@ -0,0 +1,5 @@
+[Service]
```

Comment:
This test explicitly uses a single-machine cluster...

If I didn't use a template, I'd have to use three distinct unit files to cover all cases. I don't see how this would be better?